README.md

```diff
@@ -19,3 +20,28 @@ quantitatively reason with them.
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+# Answer
+Median-of-three:
+1. On an average input, our chosen pivot is equally likely to be the $i^th$ smallest for any i = 1, 2, ... , n.
+2. With probability 1/3, our pivot will be from the middle $n$/2 elements – a good pivot.
+3. Any good pivot creates two partitions of size at most $3n/4$.
+4. We expect to pick one good pivot every three tries.
```

Comment:
But half of the elements are good pivots. Why are you only expecting to pick a good one every three tries?